src/main/java/org/example/controllers/JobRoleController.java

```diff
@@ -31,4 +33,16 @@ public Response getListOpenJobRoles() {
             return Response.serverError().build();
         }
     }
+
+    @GET
+    @Path("/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getJobRoleById(@PathParam("id") final int id) {
+        try {
+            return Response.status(HttpStatus.OK_200)
```

Comment:
For this I would copy what we did in the GetListOpenJobRoles method with Response.ok(), saves the extra import for the HTTP status stuff from Jetty. if you hit command and click on ok() up above you can dive into that method and see what it does, eventually it sets status as ` OK(200, "OK"),`

Reply:
fixed and removed the import!